pvliveconsumer/nightime.py

```diff
@@ -41,6 +41,13 @@ def make_night_time_zeros(
     if regime != "in-day":
         return gsp_yield_df
 
+    if gsp.gsp_id not in gsp_locations.index:
+        logger.warning(
+            f"Could not find gsp {gsp.gsp_id} in the gsp locations file, "
+            f"this should not happen, please check the file"
+        )
+        return gsp_yield_df
```

Comment:
If this shouldn't happen, should we not error?

Reply:
Ill remove this, I think this was a temp solution